code/game/vehicles/spacepods/equipment.dm

```diff
@@ -186,3 +187,41 @@
 /obj/item/device/spacepod_equipment/cargo/ore/removed(var/mob/user)
 	. = ..()
 	unload()
+
+/obj/item/device/spacepod_equipment/lock
+	name = "pod lock"
+	desc = "You shouldn't be seeing this"
+	icon = 'icons/goonstation/pods/ship.dmi'
+	icon_state = "blank"
+	var/mode = 0
+	var/id = null
+
+// Key and Tumbler System
+/obj/item/device/spacepod_equipment/lock/keyed
+	name = "\improper spacepod tumbler lock"
+	desc = "A locking system to stop podjacking. This version uses a standalone key."
+	icon_state = "pod_locator"
+
+/obj/item/device/spacepod_equipment/lock/keyed/New()
+	..()
+	id = rand(1, 99999)
```

Comment:
The reason I used rand() here for the ID is because real key and tumbler locks have a set number of configurations that while large, are not infinite. In other words, if you tried a key on 100000 locks and didn't find several matches, you've got yourself a miracle.

If other people think that static var sourced perfect ID numbers are the way to go, I'll certainly make the change.